mathics/builtin/assignments/assignment.py

```diff
@@ -66,17 +66,17 @@ class LoadModule(Builtin):
       <dd>'Load Mathics definitions from the python module $module$
     </dl>
     >> LoadModule["nomodule"]
-     : Python module nomodule does not exist.
+     : Problem importing Python module: nomodule.
      = $Failed
     >> LoadModule["sys"]
-     : Python module sys is not a pymathics module.
+     : Python module "sys" is not a Mathics3 module.
      = $Failed
     """
 
     name = "LoadModule"
     messages = {
-        "notfound": "Python module `1` does not exist.",
-        "notmathicslib": "Python module `1` is not a pymathics module.",
+        "notfound": """Problem importing Python module: `1`.""",
```

Comment:
If `LoadModule` produces the error `notfound`, it is because the module can not be found, not because there was a generic error in importing it. So, why the new message is better?

Reply:
There is a difference between something existing and something importing. In fact another check could be made to see if the _file_ exists and give that message.  The file may exist, but there is invalid Python or it may import modules that are not importable.

Reply:
OK, but the error we handle in importing the python module is about the existence of the library.

Reply:
"Exist" is a vague or misleading word. The library can exist in some sense but not work. When I see an error message or warning I generally want to know what was tried. What was tried is that we ran a Python kind of "import": and that did not work,.

Reply:
In Python, what we are capturing is:
``` 
>>> import amadeupmodulename
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'amadeupmodulename'
```
So, maybe "Python module `1` was not found in the system"
could be a more accurate message, isn't it?

Reply:
There is a differences here. You issued the `import` and not "not found" so it is clear that this is in response to Python "import".  In Mathics3 though, the user a priori doesn't know that LoadModule runs a Python `import`. 

So using the words Python and import suggest what kind of thing is failing. In fact it suggest trying to run a Python "import".

Reply:
What about ```Pymathics module `1` was not found in the system```? Otherwise,  it was just a suggestion. Please just keep it as you prefer.

Reply:
I have been thinking about this some more. I do listen to your suggestions...

I think we can get the best of both worlds by catching _any_  Python exception and passing back the message in the exception. In those cases where Python reports "not found" the message has that (indicating this is from an import). 

In fact, it was probably a mistake using `except ImportError` rather then `except Exception`.